### PR TITLE
[ui][update] remove static wait from 3scale afterhook

### PR DESCRIPTION
**Pull request by mcada**
//test: `@3scale,@logout-test-logout-login`

### CODE REVIEW

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by avano** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

File: utilities/pom.xml [hunk 1]
```diff
@@ -218,11 +219,17 @@
...
 		<dependency>
 			<groupId>com.google.apis</groupId>
 			<artifactId>google-api-services-calendar</artifactId>
 		</dependency>
 
+        <dependency>
+            <groupId>com.codeborne</groupId>
+            <artifactId>selenide</artifactId>
+        </dependency>
+
```
Comment (avano) [inline utilities/pom.xml]: i'm not sure if including `selenide` in the utilities is the right choice

Comment (avano) [reply utilities/pom.xml]: by the way, why? :dancer: it is not used anywhere

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

File: utilities/pom.xml (hunk 1 above)
Comment (mcada) [reply utilities/pom.xml]: its just a test, not final PR

Comment (mcada) [reply utilities/pom.xml]: I was refreshing selenide in many places to observe change of behavior locally, that was just leftover. PR is now ready, tests should PASS.

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

File: ui-tests/src/test/java/io/syndesis/qe/steps/CommonSteps.java [hunk 2]
```diff
@@ -734,7 +735,9 @@ public void set3scaleEnvVar(String value) {
             fail("Syndesis server did not start in 300s with new variable", e);
         }
 
-        WebDriverRunner.getWebDriver().navigate().refresh();
+        // new server pod means we need to reset rest route url
+        RestUtils.updateRestUrl(true);
```
Comment (avano) [inline ui-tests/src/test/java/io/syndesis/qe/steps/CommonSteps.java]: it should be necessary only to call `RestUtils.reset()` 

next time the call to `RestUtils.getRestUrl()` would create a new port forward anyway, so the `updateRestUrl()` is not needed imho

Comment (avano) [reply ui-tests/src/test/java/io/syndesis/qe/steps/CommonSteps.java]: or just make this static https://github.com/syndesisio/syndesis-qe/blob/289cbd973191fe14087cb4fc82bad3f5ee5d0ed3/utilities/src/main/java/io/syndesis/qe/bdd/CommonSteps.java#L211-L215

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mcada** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Pull request merged**